front/src/reducers/authorization/listener.ts

```diff
@@ -0,0 +1,15 @@
+import { isRejectedWithValue } from '@reduxjs/toolkit';
+
+import { type AppStartListening } from 'store/listenerMiddleware';
+
+export default function add403HttpErrorListener(startListening: AppStartListening) {
+  startListening({
+    matcher: isRejectedWithValue,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    effect: (action: any) => {
+      if (action.payload?.status === 403) {
+        window.location.href = '/403';
+      }
+    },
+  });
```

Comment:
Is there a simple way to test this? (Maybe patching editoast to send a 403 unconditionally for one request?)

Reply:
yes, you can apply this patch and try to search any Operational point in any map of application : 
```diff
diff --git a/editoast/src/views/search.rs b/editoast/src/views/search.rs
index 231aedc38..07cc0e772 100644
--- a/editoast/src/views/search.rs
+++ b/editoast/src/views/search.rs
@@ -359,7 +359,7 @@ pub async fn search(
     let conn = &mut db_pool.get().await?;
     let objects = query.load::<SearchDBResult>(conn).await?;
     let results: Vec<_> = objects.into_iter().map(|r| r.result).collect();
-    Ok(HttpResponse::Ok().json(results))
+    Ok(HttpResponse::Forbidden().json(results))
 }
 
 // NOTE: every structure deriving `Search` here might have to `#[allow(unused)]`
 ```

Reply:
Nice!